apps/condo/bin/prepare.js

```diff
@@ -8,11 +10,21 @@ async function updateAppEnvAddressSuggestionConfig (serviceName) {
     await updateAppEnvFile(serviceName, 'FAKE_FINANCE_INFO_CLIENT', 'true')
 }
 
+async function updateAppEnvFileClients (appName) {
+    await updateAppEnvFile(appName, 'FILE_UPLOAD_CONFIG', JSON.stringify({
+        clients: {
+            condo: { secret: uuid() },
+            miniapp: { secret: uuid() },
```

Comment:
can individual apps do this instead? like preparing themself by adding something in condo?